packages/flutter/lib/src/widgets/route_notification_messages.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart' show Route;
```

Comment:
This might be the import that trips up the analyzer. `widgets.dart` imports/exports this file and now this file imports `widgets.dart` causing a cycle.

Reply:
This should probably just import `navigator.dart` as a relative path.

Reply:
Done! How can I see these errors in my local before I push?

Thanks for pointing this out!

Reply:
I'm not sure. I'd expect `flutter analyze --flutter-repo` to cover this. @goderbauer ?